lib/adf_dataset.py

```diff
@@ -76,87 +78,19 @@ def set_reference(self):
             # when using a reference simulation, allow a "special" attribute with the case name:
             self.ref_case_label = self.adf.get_baseline_info("cam_case_name", required=True)
             for v in self.adf.diag_var_list:
+                self.ref_var_nam[v] = v
```

Comment:
I pulled these out the else statement because initially these weren't being populated because the check: 

```
f = self.get_reference_climo_file(v)
if f is None:
```

`f` will be none until the climo files are made, but we need ``` self.ref_var_nam``` and ```self.ref_labels``` before we get to climo file generation.